### DISCUSSION TITLE
add temporary mount point for archives

### DISCUSSION THREAD

**Discussion by renner0e** [borgbase/vorta#1864] [Ideas]
I think its really annoying having to select a mount point every time you want to mount an archive.
It would be great if there was a way for vorta to create a temporary mount point (like pikabackup).

Pikabackup does it like this for example: "/run/user/1000/pika-backup/mount/a484904f/96b748-1cecd6ae/path/to/archive"

This could also enable having a button to open the archive and not having to select a mount point. Thus reducing friction.

Love this project  👍

**Comment by jetchirag** [comment]
Related PR #1664
